underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java

```diff
@@ -71,15 +71,7 @@ public boolean supportsPath(String path) {
    * @return true if both access and secret key are present, false otherwise
    */
   private boolean addAndCheckGoogleCredentials() {
-    // TODO(binfan): remove System.getProperty as it is covered by configuration
-    String accessKeyConf = PropertyKey.GCS_ACCESS_KEY;
-    if (System.getProperty(accessKeyConf) != null && !Configuration.containsKey(accessKeyConf)) {
-      Configuration.set(accessKeyConf, System.getProperty(accessKeyConf));
-    }
-    String secretKeyConf = PropertyKey.GCS_SECRET_KEY;
-    if (System.getProperty(secretKeyConf) != null && !Configuration.containsKey(secretKeyConf)) {
-      Configuration.set(secretKeyConf, System.getProperty(secretKeyConf));
-    }
-    return Configuration.containsKey(accessKeyConf) && Configuration.containsKey(secretKeyConf);
+    return Configuration.containsKey(PropertyKey.GCS_ACCESS_KEY)
```

Comment:
that should be taken care by the init process of `Configuration` which loads system property into `Configuration`